app/templating/summary/formatters/abstract_formater.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABCMeta, abstractmethod
+
+
+class AbstractFormatter(metaclass=ABCMeta):
+
+    @staticmethod
+    @abstractmethod
+    def format(schema_answers, state_answers, user_answer):
```

Comment:
Are we able to make use of keyword arguments here in order to allow child classes to omit variables that aren't used in formatting?